sdk/azcore/internal/pollers/util.go

```diff
@@ -25,18 +25,19 @@ import (
 const (
 	StatusSucceeded  = "Succeeded"
 	StatusCanceled   = "Canceled"
+	StatusCancelled  = "Cancelled"
```

Comment:
This violates the [ARM RPC spec](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#provisioningstate-property) and is incorrectly spelled.  If services are returning this, then they need to be fixed.

Reply:
I think it makes sense to change the ARM RPC spec since a large percentage of existing specifications use that valid spelling.

Reply:
@JeffreyRichter @mikekistler, Should there be a sweep through all the REST specs to correct them to follow the RPC spec?  Adding support for something that violates the RPC spec is not something we want to do.

Reply:
Ultimately, the ARM team will need to make this call.

Reply:
I did the analysis. There are only 8 control plane services affected. The one I manage, plus 7 others. All log bugs for each.